drivers/ili9341/Kconfig

```diff
@@ -13,3 +13,44 @@ config MODULE_ILI9341
     select MODULE_PERIPH_SPI
     select MODULE_PERIPH_GPIO
     select MODULE_XTIMER
+
+menuconfig KCONFIG_USEMODULE_ILI9341
+    bool "Configure ILI9341 driver"
+    depends on USEMODULE_ILI9341
+    help
+        Configure the ILI9341 display driver using Kconfig.
+
+if KCONFIG_USEMODULE_ILI9341
+
+config ILI9341_GVDD
+    int "GVDD voltage level (in millivolts)"
+    default 4800
+    help
+        Configure GVDD level, which is a reference level for the VCOM level and
+        the grayscale voltage level.
```

Comment:
If I'm not mistaken the datasheet allows values between 3000 and 6000 for this right? If that is the case a range should be added.

Reply:
Yes. The ili9341 allows it. Shall update :)